repack.go

```diff
@@ -102,7 +102,8 @@ func Repack(engineExt casext.Engine, tagName string, bundlePath string, meta Met
 			return err
 		}
 	} else {
-		reader, err := layer.GenerateLayer(fullRootfsPath, diffs, &meta.MapOptions)
+		packOptions := layer.PackOptions{MapOptions: meta.MapOptions}
```

Comment:
While the whiteout type is still internal, maybe we should include the whiteout type in the extracted image's `Meta`...